radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/P2PModule.java

```diff
@@ -180,6 +193,12 @@ protected void configure() {
         Duration.ofSeconds(10));
   }
 
+  @Provides
+  public PeersView peersView(
+      PeerManagerPeersView peerManagerPeersView, ProxiedPeersView proxiedPeersView) {
+    return PeersView.combine(peerManagerPeersView, proxiedPeersView);
```

Comment:
perhaps we could consider the `useProxies` p2p config here, like so:
```
return proxyConfig.useProxies
  ? PeersView.combine(peerManagerPeersView, proxiedPeersView)
  : peerManagerPeersView;
```

Reply:
Done